gratipay/billing/payday.py

```diff
@@ -408,23 +408,22 @@ def payout(self):
         log("Starting payout loop.")
         participants = self.db.all("""
             SELECT p.*::participants
-              FROM participants p
-             WHERE balance > 0
-               AND ( SELECT count(*)
-                       FROM exchange_routes r
-                      WHERE r.participant = p.id
-                        AND network = 'balanced-ba'
-                   ) > 0
+              FROM exchange_routes r
+              JOIN participants p ON p.id = r.participant
```

Comment:
I think replacing `exchange_routes` with `current_exchange_routes` in both places should solve the problem.